stdlib/runtime/wasi.gr

```diff
@@ -26,6 +26,15 @@ import foreign wasm clock_time_get: (WasmI32, WasmI64, WasmI32) -> WasmI32 from
 import foreign wasm path_open : (WasmI32, WasmI32, WasmI32, WasmI32, WasmI32, WasmI64, WasmI64, WasmI32, WasmI32) -> WasmI32 from "wasi_snapshot_preview1"
 import foreign wasm fd_read : (WasmI32, WasmI32, WasmI32, WasmI32) -> WasmI32 from "wasi_snapshot_preview1"
 import foreign wasm fd_pread : (WasmI32, WasmI32, WasmI32, WasmI64, WasmI32) -> WasmI32 from "wasi_snapshot_preview1"
+/**
+ * Invokes the `fd_write` system call.
+ *
+ * @param file_descriptor - The file descriptor to write to
+ * @param iovs - The pointer to the array of iovs to write
+ * @param iovs_len - The length of the array of iovs
+ * @param nwritten - Where to store the number of bytes written
+ * @return The number of bytes written
+ */
```

Comment:
Technically not the correct format for graindoc, but I doubt we want to run graindoc on these, yeah?

Reply:
Actually, I don't think it iterates foreigns anyway.

Reply:
🤷‍♂️ 
@phated Should I change it? If so, can you suggest a change?

Reply:
```suggestion
/**
 * Invokes the `fd_write` system call.
 *
 * @param file_descriptor: The file descriptor to write to
 * @param iovs: The pointer to the array of iovs to write
 * @param iovs_len: The length of the array of iovs
 * @param nwritten: Where to store the number of bytes written
 * @return The number of bytes written
 */
```
We use colon as a separator, but like I mentioned above, this won't be documented by graindoc because we don't iterate foreign statements afaik (and the signatures don't have parameter names to match on).